app/models/exam/passing_criterion.rb

```diff
@@ -0,0 +1,53 @@
+class Exam::PassingCriterion
+
+  attr_reader :value
+
+  def initialize(value)
+    @value = value
+  end
+
+  def type
+    self.class.name.demodulize.underscore
```

Comment:
Here you could use mumukit-core `as_variable_name`

Reply:
Mmm I think it wouldn't work as it doesn't demodulize it :confounded: